.travis.yml

```diff
@@ -1,12 +1,12 @@
-dist: trusty
-sudo: false
+dist: xenial
```

Comment:
According to the [docs](https://docs.travis-ci.com/user/reference/linux), xenial is already the default version. Should we remove this or bump up to `bionic`?

Reply:
I am pinning to xenial so that when they change the defaults again later, stuff doesn't break.